core/forms.js

```diff
@@ -2,7 +2,9 @@ define('core/forms', ['core/z'], function(z) {
 
     function checkValid(form) {
         if (form) {
-            $(form).filter(':not([novalidate])').find('button[type=submit]').attr('disabled', !form.checkValidity());
+            var currentForm = $(form);
```

Comment:
We tend to name jquery elements as `$currentForm` etc. I would also do another `var $textArea = $currentForm.find('textarea')` so we're not doing a jquery lookup every time.